rules/windows/process_creation/proc_creation_win_powershell_comobject_msi.yml

```
@@ -0,0 +1,45 @@
+title: Obfuscated PowerShell Silent MSI Install via WindowsInstaller COM
```

```suggestion
title: Obfuscated PowerShell MSI Install via WindowsInstaller COM
```
since we are not specifying the UILevel to be 2